docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -100,67 +100,67 @@ After attempting to deploy {WebNameTitle} on OpenShift, the Pod is unable to sta
+* Increase the quota for the OpenShift project. For additional information on OpenShift quotas, see link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[Quotas and Limit Ranges] and link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[Setting Limit Ranges].
 * Reduce the requested resources for the {WebName} OpenShift project. It is recommended to have at least 2 CPUs and 4 GB of memory for the project.
 
 Once the request is within the available quota, attempt the deployment once again.
 
-.Pod takes longer than 600 seconds to become available
+.A pod takes longer than 600 seconds to become available
 
-After attempting a deployment, the `-deploy` Pods time out and report the following error.
+After attempting a deployment, the `-deploy` pods time out and report the following error:
 
 [source,options="nowrap",subs="+quotes"]
 ----
 error: update acceptor rejected mta-web-console-executor-1: Pods for rc 'mta/<POD_NAME>' took longer than 600 seconds to become available
 ----
 
-These errors appear after the Pods time out and are placed into an error state.
+These errors appear after the pods time out and are placed into an error state.
 
 *What it means:*
 
-The deployment Pods are unable to successfully launch the Pods.
+The `-deploy` pods are unable to successfully launch the pods.
 
 This error can be caused by a number of sources, with the following being the most common:
 
-* The OpenShift instance is currently out of resources to deploy the Pod in a timely manner.
+* The OpenShift instance is currently out of resources to deploy the pod in a timely manner.
 * The images were unable to be successfully pulled from the registry.
 
 *How to resolve it:*
 
-Attempt the deployment again, and view the logs and events of the non deployment Pods while they are being created. These messages will provide context to the underlying errors resulting in the deployment Pod timeouts.
+Attempt the deployment again, and view the logs and events of the non-deployment pods while they are being created. These messages provide context to the underlying errors resulting in the `-deploy` pod timeouts.
 
-* If the OpenShift instance runs out of resources, link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[analyze your cluster's capacity]. Once the capacity has increased, or there are fewer jobs executing, attempt the deployment once again.
+* If the OpenShift instance is out of resources, follow the instructions in link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-resource-levels.html[Analyzing Cluster Capacity] to determine the cluster capacity. Once the capacity has increased, or there are fewer jobs executing, attempt the deployment once again.
```

Comment:
This is weird. If the cluster is running out of resources, wouldn't an error message like the one above (0/9 nodes are available: 4 Insufficient cpu, 4 MatchNodeSelector, 9 Insufficient memory) also appear?